README.md

```diff
@@ -3,36 +3,55 @@
-[![Codacy Coverage Badge](https://api.codacy.com/project/badge/Coverage/79bf6932c2e844eea15d0fb1ed7e415c)](https://www.codacy.com/app/ESMValGroup/ESMValCore?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ESMValGroup/ESMValCore&amp;utm_campaign=Badge_Coverage)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/79bf6932c2e844eea15d0fb1ed7e415c)](https://www.codacy.com/app/ESMValGroup/ESMValTool?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ESMValGroup/ESMValTool&amp;utm_campaign=Badge_Grade)
+[![Codacy Badge](https://api.codacy.com/project/badge/Coverage/5d496dea9ef64ec68e448a6df5a65783)](https://www.codacy.com/app/ESMValGroup/ESMValCore?utm_source=github.com&utm_medium=referral&utm_content=ESMValGroup/ESMValCore&utm_campaign=Badge_Coverage)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/5d496dea9ef64ec68e448a6df5a65783)](https://www.codacy.com/app/ESMValGroup/ESMValCore?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ESMValGroup/ESMValCore&amp;utm_campaign=Badge_Grade)
 [![Docker Build Status](https://img.shields.io/docker/build/esmvalgroup/esmvaltool.svg)](https://hub.docker.com/r/esmvalgroup/esmvaltool/)
 [![Anaconda-Server Badge](https://anaconda.org/esmvalgroup/esmvalcore/badges/installer/conda.svg)](https://conda.anaconda.org/esmvalgroup)
 
 
-ESMValTool Core: core functionalities for the ESMValTool, a community diagnostic and performance metrics tool for routine evaluation of Earth system models in CMIP
+ESMValTool Core: core functionalities for the ESMValTool, a community
+diagnostic and performance metrics tool for routine evaluation of Earth System
+Models in CMIP.
 
 # Getting started
-This is the development branch for ESMValTool Core. ESMValTool version 2 is under rapid development, an installation from source is recommended at the moment.
-
-## Installing from source [recommended]
-Please see [CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/development/CONTRIBUTING.md) for instructions on installing ESMValTool from source.
 
 ## Installing from Anaconda
 The Anaconda package can be found on [ESMValGroup Anaconda Channel.](https://anaconda.org/ESMValGroup)
 
-If you already installed Anaconda, you can install ESMValTool by running:
+If you already have
+[Anaconda installed](https://conda.io/projects/conda/en/latest/user-guide/install/index.html),
+you can install ESMValTool Core by running:
 ```
 conda install -c esmvalgroup -c conda-forge esmvalcore
 ```
 
+## Installing from source
+If you need the very latest features, please see
+[CONTRIBUTING.md](https://github.com/ESMValGroup/ESMValCore/blob/development/CONTRIBUTING.md)
+for instructions on installing ESMValTool Core from source.
+
+## Using ESMValTool Core as a Python library
+The ESMValTool Core package provides various functions for:
+- Finding data in a directory structure typically used for CMIP data
```

Comment:
```suggestion
- Finding and reading data in a directory structure typically used for CMIP data
```

Reply:
Do we provide functions for reading data? Which ones?

Reply:
The Iris ones.

Reply:
So iris provides this functionality, right? Not us?

Reply:
Well, then we should also remove this sentence "ESMValTool preprocessor functions for e.g. regridding, vertical interpolation, ...", since we use Iris for that.

What I wanted to point out, is that the core is used for finding, reading and preprocess the data.
Reading, in our case, also includes more complex functionalities, like concatenating multiple files and extracting a time range.

Reply:
> Well, then we should also remove this sentence "ESMValTool preprocessor functions for e.g. regridding, vertical interpolation, ...", since we use Iris for that.

I disagree, our preprocessor functions offer something extra compared to just iris. Indeed they use iris, but they are not only that.

> Reading, in our case, also includes more complex functionalities, like concatenating multiple files and extracting a time range.

I'll add those to the list of preprocessor functions, since that is what they are

Reply:
Ok.